backends/p4tools/scripts/tools/run-clang-format

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+
+args=$(getopt -l "write" -o "wh" -- "$@")
```

Comment:
can we give a .sh suffix to this file to make it clear that it's a shell script?